example-config.el

```diff
@@ -0,0 +1,31 @@
+;;; example-config.el -- Example Rational Emacs user customization file -*- lexical-binding: t; -*-
+
+;;; Commentary:
+;;
+;; Rational Emacs supports user customization through a `config.el' file
+;; similar to this one.  You can copy this file as `config.el' to your
+;; Rational Emacs configuration directory as an example.
+;;
+;; In your configuration you can set any Emacs configuration variable, face
+;; attributes, themes, etc as you normally would.
+;;
+;; See the README.org file in this repository for additional information.
+
+;;; Code:
+(require 'rational-defaults)
+(require 'rational-screencast)
+(require 'rational-ui)
+(require 'rational-editing)
+(require 'rational-evil)
+(require 'rational-completion)
+(require 'rational-windows)
+
+;; Set further font and theme customizations
+(set-face-attribute 'default nil
```

Comment:
A better way would be to define the default face in early-config.el. The reason is, it gets set and the frame size can be calculated based on the font size before it is rendered, which avoids an annoying resizing behavior. Also why we turn off the toolbar and menubar there (emacs avoids having to compute a size for a frame containing them only to turn them off once the frame is rendered, which then causes the frame to suddenly shrink).

Reply:
Yeah, you make a good point.  It's probably better to show an example of setting the font in `early-config.el`.  That would have solved the problem we were having on stream yesterday where my font was tiny until the full config loaded up :)